assets/scripts/auth/api-user.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const app = require('../app-data');
+
+const signUp = (success, failure, data) => {
```

Comment:
Modified accordingly. signOut did not have data in the ajax so I added that as well.